packages/client-sdk-web/src/index.ts

```diff
@@ -95,6 +95,16 @@ import {
   TokenScope,
   ExpiresIn,
   ExpiresAt,
+  CachePermissionOptions,
+  TopicPermissionOptions,
+  CacheName,
+  isCacheName,
+  TopicName,
+  isTopicName,
```

Comment:
I would guess that we don't need these `isCacheName` and `isTopicName` to be considered part of the public API, so I wouldn't export them from this index. wdyt @krispraws ?

Reply:
Agree. I exported them by mistake in the nodejs index. I will fix that.